contents/architecture/RESTArchitecturalStyle.md

```diff
@@ -0,0 +1,226 @@
+# REST Architectural Style
+
+Author: [Wen Xin](https://github.com/wenmogu)
+
+This book chapter introduces the REST architectural style, which is the guideline of development for the Web we use today. The following content is a summary of selected parts from the famous [Roy Fielding's paper about REST](https://www.ics.uci.edu/~fielding/pubs/dissertation/fielding_dissertation.pdf) with my own understanding, which I hope can serve as a relatively brief, easy-to-understand and at the same time detailed introduction to REST.
+
+**Requirement of Web development --- Background of REST architectural style**
```

Comment:
You might want to consider using headers instead (e.g. `##`) to better distinguish section headers from section content...Right now it's the same size as everything else in the chapter. Same goes for all the section headings below in this chapter.

Reply:
Also, is it possible to shorten this header to maybe something like just "Background of REST architectural style"?